docs/server/source/production-deployment-template/node-on-kubernetes.rst

```diff
@@ -866,7 +866,7 @@ To test the NGINX instance with HTTPS and 3scale integration:
 
    $ wsc -er wss://<cluster-fqdn>/api/v1/streams/valid_transactions
 
-   $ curl -X GET https://<cluster-fqdn>
+   $ curl -X GET http://<cluster-fqdn>:27017
```

Comment:
Now that we're using MongoDB authentication for all communications with MongoDB, does MongoDB still respond with the "It looks like you are trying to access MongoDB over HTTP..." message, or does it say something about authentication being required?